SmartDeviceLinkTests/RPCSpecs/RequestSpecs/SDLAlertSpec.m

```diff
@@ -76,7 +76,7 @@
                                                    SDLRPCParameterNameDuration:@4357,
                                                    SDLRPCParameterNamePlayTone:@YES,
                                                    SDLRPCParameterNameProgressIndicator:@NO,
-                                                   SDLRPCParameterNameSoftButtons:[NSNull null]},
+                                                   SDLRPCParameterNameSoftButtons:@[[NSNull null]]},
```

Comment:
This should be in a separate test for bad data, not in a test for good data.

Reply:
please see 260159e